spec/scry/completion_provider_spec.cr

```diff
@@ -1,38 +1,133 @@
 require "../spec_helper"
 
 module Scry
+  BOOLEAN_METHODS = %w(!= & == ^ clone hash to_json to_s to_s to_yaml |)
+  INT32_METHODS   = %w(- clone popcount)
+
+  private macro it_completes(code, with_labels = nil, file = __FILE__, line = __LINE__)
+
+    it "completes #{{{code}}.dump}" do
+      {% if with_labels %}
+        %expected = {{with_labels}}
+      {% else %}
+        %expected = _expected_labels_
+      {% end %}
+
+      %code = {{code}}
+      cursor_location = [%code.lines.size, %code.size]
```

Comment:
You'd better use a tuple here (no need to create a full array for that), so `{%code.lines.size, %code.size}`

Reply:
I tried to do this initially but was getting a syntax error I didn't understand.
Went back and tried it again and noticed the issue was that `{%code` would be interpreted as macro expression or something, had to add a space between `{` and `%` for it to work :( 
Thanks for the review 👍